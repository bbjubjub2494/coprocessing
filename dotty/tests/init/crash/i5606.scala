object Test extends App {

  def (f: A => B) `$`[A, B](a: A): B = f(a)

  assert((((a: Int) => a.toString()) `$` 10) == "10")

  def g(x: Int): String = x.toString

  assert((g `$` 10) == "10")

  val h: Int => String = _.toString

  assert((h `$` 10) == "10")
}
