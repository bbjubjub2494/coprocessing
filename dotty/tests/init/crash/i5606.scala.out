-- Error: /Users/fliu/Documents/scala-init-checker/tests/crash/i5606.scala:1:7 -----------------------------------------
1 |object Test extends App {
  |       ^
  |       Leaking of this. Calling trace:
  |        -> object Test extends App {	[ /Users/fliu/Documents/scala-init-checker/tests/crash/i5606.scala:1 ]
one error found
